### ISSUE TITLE
SAPS in MIMIC-IV

### ISSUE THREAD

**Issue by VoyagerWSH** [MIT-LCP/mimic-code#1432]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Hi! I am wondering if the SAPS-II scores for ICU visits are calculated in the MIMIC-IV dataset, and which module/table are they included in? If they are not currently included, will future versions of MIMIC-IV include mortality probability scores (e.g., SAPS, APACHE, SOFA) for ICU visits? Thank you so much for the help!

**Comment by snowgato** [comment]
Hi,
You can access saps II scores in mimiciv_derived dataset, that contains sapsii table with saps ii scores.

**Comment by VoyagerWSH** [comment]
Thanks for the response! I am wondering where I can find information on the mimiciv_derived dataset? I navigated through the [module documentations for MIMIC-IV](https://mimic.mit.edu/docs/iv/modules/icu/) but haven't found this dataset. Thanks!

**Comment by snowgato** [comment]
You can access mimiciv-derived dataset using Bigquery in physionet-data project. Sapsii table fields names are self-explanatory.
Otherwise, if you want to have a look at how the table is built, you should download the files that enable you to build locally the database. You'll find there a directory named mimic-code/mimic-iv/concepts/score containing sapsii.sql, the script that builds the sapsii table.

**Comment by alistairewj** [comment]
Yeah, the tables in the `mimiciv_derived` schema don't have any online documentation, but you can read through the source code [in the concepts folder](https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv/concepts), and the idea behind it is described in [The MIMIC Code Repository paper](https://doi.org/10.1093/jamia/ocx084). Probably we should add a MIMIC-IV derived section in the online docs, but not sure when we'll get time for that!
